twisted/python/compat.py

```diff
@@ -756,6 +759,25 @@ def _coercedUnicode(s):
 
 
 
+def _maybeMBCS(s):
+    """
+    Convert the string C{s} to a L{unicode} string, if required.
+
+    @param s: The string to convert.
+    @type s: L{bytes} or L{unicode}
+
+    @return: The string, decoded using MBCS if needed.
```

Comment:
Just that it raises `UnicodeDecodeError` if passed a bytestring that can't be decoded using the local encoding, and also that the local MBCS encoding may well be incapable of representing reasonable strings because it's tied to the charset.  If in fact I am understanding MBCS properly - every time I touch Windows I have to remind myself how all this legacy stuff works :-\